HIP/hip-1.md

```diff
@@ -151,26 +151,28 @@ Each HIP should have the following parts/sections:
 4. Rationale -- The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages.
 
     The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during the discussion.
+    
+5. User stories -- Provide a list of "user stories" to express how this feature, functionality, improvement, or tool will be used by the end user. Template for user story: “As (user persona), I want (to perform this action) so that (I can accomplish this goal).”
```

Comment:
We should also update the [HIP template](https://github.com/hashgraph/hedera-improvement-proposal/blob/master/hip-0000-template.md).

Reply:
+1 did an update in a PR here: https://github.com/hashgraph/hedera-improvement-proposal/pull/124

Reply:
> See my note about DCO

Ah, thanks — having trouble figuring out how to DCO (haven't used that feature before). Where do I add my DCO? Can I do this manually by just appending "Signed-off-by: Brady Gentile brady@hedera.com" to my comment?

Reply:
You can just copy and paste the commands [here](https://github.com/hashgraph/hedera-improvement-proposal/pull/123/checks?check_run_id=3174345013) on the command line.

Reply:
For new commits, just always run `git commit --signoff`.